### DISCUSSION TITLE
Writing a custom solution to call WC RPC API directly?

### DISCUSSION THREAD

**Discussion by grigb** [WalletConnect/walletconnect-monorepo#2611] [Developer Support]
I need to call WalletConnect from C# (Unity) and vanilla JS (external sites I don't host).

I can't use the pre-written solution because I'm using the Chia blockchain.

Can I just call RPC calls directly to the WC server?

Are there any examples of this?

**Comment by Blackcode** [comment]
Still work in progress but give it a try...
https://github.com/Blackcode/ChiaWalletConnect.dotnet
